crates/zed/src/main.rs

```diff
@@ -1514,3 +1462,12 @@ fn dump_all_gpui_actions() {
     )
     .unwrap();
 }
+
+#[cfg(windows)]
+fn check_for_conpty_dll() {
+    use windows_sys::{Win32::System::LibraryLoader::LoadLibraryW, w};
```

Comment:
NIT: I recall, the previous version was able to read this dll from a custom path?

That bit made sense, IMO? 
If the person disables installation of Zed's bundled dll, there could be a centralized place where those reside?